.github/workflows/go_lint.yml

```diff
@@ -2,7 +2,7 @@ name: Go lint
 
 on:
   push:
-    branches: [ master ]
+    branches: [ main ]
```

Comment:
@pierluca @jbsv - not sure if you want to switch to `main` branch instead of `master`. We switched all our repos to `main`: https://www.theserverside.com/feature/Why-GitHub-renamed-its-master-branch-to-main